airflow/providers/docker/operators/docker.py

```diff
@@ -157,7 +158,7 @@ def __init__(
         tls_ssl_version: Optional[str] = None,
         tmp_dir: str = '/tmp/airflow',
         user: Optional[Union[str, int]] = None,
-        volumes: Optional[List[str]] = None,
+        mounts: Optional[List[Mount]] = None,
         entrypoint: Optional[Union[str, List[str]]] = None,
```

Comment:
It looks like breaking change. Can you add note about it to provider docs? Are you looking a example? We have one example in Google provider: https://github.com/apache/airflow/blob/master/airflow/providers/google/CHANGELOG.rst